CHANGELOG.rst

```diff
@@ -1,9 +1,17 @@
 Changelog for Mantarray Frontend Components
 ===========================================
 
-1.4.5 (unreleased)
+1.4.6 (unreleased)
 ------------------
 
+Changed:
+^^^^^^^^
+- Styling of stimulation warning widget. Background is now red.
```

Comment:
stimulation running widget* 
and the changelog entry below for 1.4.5 could be less specific of placement since that's not relevant to this repo, just addition of the Stimulationrunningwidget